clusters/k8s-infra-dev-cluster-turnup/10-clusterconfiguration.tf

```diff
@@ -89,14 +102,24 @@ resource "google_container_cluster" "cluster" {
     }
   }
 
+  // Enable google-groups for RBAC
+  authenticator_groups_config {
+    security_group = "gke-security-groups@kubernetes.io"
+  }
+
+  // Enable workload identity for GCP IAM
+  workload_identity_config {
```

Comment:
Does enabling this on the cluster mean that we *have* to use it?

Reply:
I don't think so.  It's totally opt-in.  Additionally, I see that it is incompatible with metadata concealment, so we should probably disable that flag too.